diff --git a/dist/edge/index.d.ts b/dist/edge/index.d.ts
--- a/dist/edge/index.d.ts
+++ b/dist/edge/index.d.ts
@@ -174,9 +174,10 @@
     googleCredentials?: GoogleCredentials;
 }
 declare function createVertex(options?: GoogleVertexProviderSettings): GoogleVertexProvider;
+declare function createVertexWithoutCredentials(options?: GoogleVertexProviderSettings$1): GoogleVertexProvider;
 /**
  * Default Google Vertex AI provider instance.
  */
 declare const vertex: GoogleVertexProvider;
 
-export { type GoogleVertexProvider, type GoogleVertexProviderSettings, createVertex, vertex };
+export { type GoogleVertexProvider, type GoogleVertexProviderSettings, createVertex, createVertexWithoutCredentials, vertex };
diff --git a/dist/edge/index.d.mts b/dist/edge/index.d.mts
--- a/dist/edge/index.d.mts
+++ b/dist/edge/index.d.mts
@@ -174,9 +174,10 @@
     googleCredentials?: GoogleCredentials;
 }
 declare function createVertex(options?: GoogleVertexProviderSettings): GoogleVertexProvider;
+declare function createVertexWithoutCredentials(options?: GoogleVertexProviderSettings$1): GoogleVertexProvider;
 /**
  * Default Google Vertex AI provider instance.
  */
 declare const vertex: GoogleVertexProvider;
 
-export { type GoogleVertexProvider, type GoogleVertexProviderSettings, createVertex, vertex };
+export { type GoogleVertexProvider, type GoogleVertexProviderSettings, createVertex, createVertexWithoutCredentials, vertex };
diff --git a/dist/edge/index.js b/dist/edge/index.js
--- a/dist/edge/index.js
+++ b/dist/edge/index.js
@@ -21,6 +21,7 @@
 var index_exports = {};
 __export(index_exports, {
   createVertex: () => createVertex2,
+  createVertexWithoutCredentials: () => createVertex,
   vertex: () => vertex
 });
 module.exports = __toCommonJS(index_exports);
diff --git a/dist/edge/index.mjs b/dist/edge/index.mjs
--- a/dist/edge/index.mjs
+++ b/dist/edge/index.mjs
@@ -1594,6 +1594,7 @@
 var vertex = createVertex2();
 export {
   createVertex2 as createVertex,
+  createVertex as createVertexWithoutCredentials,
   vertex
 };
 //# sourceMappingURL=index.mjs.map