diff --git a/dist/silver-fleece.umd.js b/dist/silver-fleece.umd.js
index a47a9eafdca9fcbb2c0495a03ce312fdd4420d20..e68506cda5f39995ed9ee660bb54bec2ffd47963 100644
--- a/dist/silver-fleece.umd.js
+++ b/dist/silver-fleece.umd.js
@@ -392,6 +392,9 @@
             return "[ ".concat(elements.join(', '), " ]");
         }
         else if (type === 'object') {
+            if (value === null) {
+                return 'null';
+            }
             var keys = Object.keys(value);
             var properties = keys.map(function (key) {
                 return stringifyProperty(key, value[key], indentation + indentString, indentString, newlines);
